dev-tools/omdb/src/bin/omdb/oxql.rs

```diff
@@ -2,38 +2,172 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
-//! OxQL shell.
+//! omdb OxQL shell for interactive queries on metrics/timeseries.
```

Comment:
See the note on the `oximeter_db::oxql::oxql_shell()` method below, but I think we should put this back into the `oximeter-db` crate.

Reply:
I do believe we should still remove the oxql feature-flag though (instead of bubbling it up). Thoughts @bnaecker?

Reply:
Actually, moving the shell pieces into oximeter-db lib, makes this not a thing.